pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -256,17 +269,150 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 
 	log.Functionf("Starting %s", agentName)
 
+	// Initialize zedagent context.
+	zedagentCtx := newContext()
```

Comment:
Should we name the function newZedafentContext() to reduce confusion with the golang Context?

Reply:
Makes sense, done